edge_computing/image-base-install/ibi-understanding-image-based-install.adoc

```diff
@@ -0,0 +1,105 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="ibi-image-based-install_{context}"]
+= Understanding an image-based installation for {sno} clusters
+include::_attributes/common-attributes.adoc[]
+:context: ibi-understanding-image-based-install
+
+toc::[]
+
+Image-based installations streamline the deployment process for {sno} clusters, reducing both installation and deployment times. By leveraging an image-based approach, you can preinstall configured and validated instances of {sno} on target hosts. These prepared hosts can be rapidly reconfigured and deployed at the far edge of the network with minimal intervention.
+
+== Understanding image-based installation and deployment for {sno} clusters
+
+Service providers continue to deploy their infrastructure at the far edge of the network. However, this presents challenges in terms of the cost, time, and expertise needed for installing and deploying {sno} clusters at remote sites on an enterprise scale.
+
+At the far edge of the network, challenges such as low bandwidth and high latency can increase installation times and complicate {sno} cluster deployments. These circumstances require experienced technicans to spend considerable time at remote sites, overseeing the installation and deployment of a {sno} cluster.
+
+An image-based installation and deployment overcomes the deficiencies of this workflow by separating the installation and deployment stages for {sno} clusters. Using an image-based installation, you can preinstall multiple hosts with {sno} at a central site, such as a service depot or a factory.
+You can also validate the base configuration for these hosts and leverage the image-based approach to perform reproducible factory installs at scale using a single container image.
+The preinstalled and validated hosts are shipped to a remote site where they can be rapidly reconfigured and deployed in a matter of minutes.
+
+.PLACEHOLDER - Overview of an image-based installation and deployment
+image::../images/ibi-workflow.png[][Overview of an image-based installation and deployment]
+
+=== Image-based installation for {sno} clusters
+Using the {lcao}, you can generate an OCI container image that encapsulates an instance of a {sno} cluster. This image is derived from a dedicated cluster that you can configure with the target {product-title} version.  
+
+You can reference this image in a live installation ISO to consistently preinstall configured and validated instances of {sno} to multiple hosts. This approach enables the preparation of hosts at a central location, for example in a factory or service depot, before shipping the preinstalled hosts to a remote site for rapid reconfiguration and deployment. 
+
+The following is a high-level overview of the image-based installation process:
+
+* Generate an image from a {sno} cluster.
```

Comment:
We use "image" a lot, maybe it's worth differentiating when we're talking about an OCI/container image versus an ISO image in more places.

Reply:
Good point - I'll try to use either image **or** install/config ISO. Later I use seed image but this concept hasn't been introduced yet at this point in the docs. Thanks